### ISSUE TITLE
Accessibitly: Keyboard trap within  (Featured Datasets) tables on the page

### ISSUE THREAD

**Issue by Dawnzar** [CityofOttawa/Ottawa-ckan#22]
WCAG 2.1.2 - Level (A) Violation - No Keyboard Trap
1. Go to: data.ottawa.ca
2. When tabbing to the tables the user becomes trapped within, and must be able to tab throughout all functionality on the page. You can shift+tab backwards but unable to proceed forward.
![featured dataset tables keyboard lock](https://cloud.githubusercontent.com/assets/5271253/8010097/6349c9b2-0b7b-11e5-97e6-00a41b051218.JPG)
